inst/extdata/child/intro-emissions_profile.Rmd

```diff
@@ -1,35 +1,43 @@
-at high risk, medium risk or low risk due to the products' relative greenhouse
-gas (GHG) emissions. The assessment is first performed on a product-level and
-can then be aggregated to the company-level.
+The "emissions profile" measures the absolute GHG emissions of a product in
+comparison to a chosen benchmark of products. The assessment is first performed
+on a product-level and can then be aggregated to the company-level. The profile
+is expressed as the share of a company's products that are in a "high",
+"medium", or "low" emission profile category, based on the comparison to the
+benchmark. A higher emission profile indicates a larger impact on climate change
+compared to the benchmark and can therefore also be interpreted as one climate
+risk component to assess products, companies, or loan portfolios.
 
-The "emissions profile" measures the relative GHG emissions per product. As a
-default option, each product is compared to the GHG emissions of every other
-product. Products with higher GHG emissions are considered to have a higher
-emission profile and hence also face a higher climate risk.
+```{r include=FALSE}
+bm <- tiltIndicator:::flat_benchmarks(tiltIndicator:::example_products())
+bm <- paste0("`", bm, "`")
+```
 
-After identifying the GHG emissions for each product, the products are ranked
-according to their carbon footprint. The ranking method is explained in more
-detail in the
-[Thresholds](https://2degreesinvesting.github.io/tiltIndicator/articles/tiltIndicator.html#thresholds)
-section. In short, products with relatively low GHG emissions are assigned to
-the category "low", products with relatively high GHG emissions to the category
-"high" and the products in between to the category "medium".
+The "emission profile" is calculated in these steps: 
 
-Note that for the ranking and categorization, we apply different benchmarks.
-Products are compared to (i) all other products, to (ii) products within the
-same tilt sector, to (iii) products within the same ISIC 4 digit sector, to (iv)
-products measured in the same unit, and a combination of (iv) with either (i),
-(ii) or (iii).
+* The relative GHG emissions per product are collected from a
+Life-Cycle-Analysis (LCA) database by matching the products from our company
+dataset to the products from the LCA dataset.
+* All products are ranked according to their GHG emissions. 
+* The products are grouped by the following benchmarks:
+  1. `r bm[[1]]`: All products.
+  1. `r bm[[2]]`: All products within the same ISIC 4 digit code (example: 0112 Growing of rice).
+  1. `r bm[[3]]`: All products within the same tilt sector (example: agriculture).
+  1. `r bm[[4]]`: All products with the same unit (example: kg).
+  1. `r bm[[5]]`: All products with the same unit within the same ISIC 4 digit section (example: kg + 0112 Growing of rice).
+  1. `r bm[[6]]`: All products with the same unit within the same tilt sector (example: kg + agriculture).
```

Comment:
@Tilmon FYI here's the solution I came up with. The output is now in sync with the code. It's fragile but I it's better to get a compilation error than to compile silently with outdated information.